common/src/main/java/org/wso2/testgrid/common/config/ScenarioConfig.java

```diff
@@ -127,7 +126,7 @@ public void setTestType(String testType) {
         this.testType = testType;
     }
 
-    private Map<String, String> inputParameters = new HashMap<>();
+    private HashMap inputParameters = new HashMap<>();
```

Comment:
Shall we change the type to Map<String, Object> ?

`private Map<String, Object> inputParameters = new HashMap<>();`